public/javascripts/continuousPlay.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-unused-vars */
+
+//page numeration
+let pageNum = parseInt(localStorage.getItem("pageNum"));
+if (pageNum) {
+  console.log("there is a page num");
+  document.getElementById("page-counter").innerText = `${pageNum} of 20`;
```

Comment:
Let's try to reduce repetition here for DOM queries

Reply:
Will try to reduce them as much as i can